src/services/CalcFactor.jl

```diff
@@ -240,11 +224,11 @@ function CommonConvWrapper( usrfnc::T,
                             res::AbstractVector{<:Real}=zeros(zDim),
                             threadmodel::Type{<:_AbstractThreadModel}=MultiThreaded,
                             inflation::Real=3.0,
-                            vartypes=typeof.(getVariableType.(factormetadata.fullvariables)),
+                            vartypes=DataType[],
```

Comment:
Why not just:
```suggestion
                            vartypes=typeof.(getVariableType.(fullvariables)),
```

Reply:
julia> typeof.(getVariableType.(DFGVariable[]))
Any[]

Reply:
But it should be done better

Reply:
done